content/en/universal_service_monitoring/_index.md

```diff
@@ -63,6 +63,36 @@ If you have feedback about what platforms and protocols you'd like to see suppor
 Enable Universal Service Monitoring in your Agent by using one of the following methods depending on how your service is deployed and your Agent configured:
 
 {{< tabs >}}
+{{% tab "Operator" %}}
```

Comment:
1. Since it is not yet "stable" I'd prefer to move it to be after `Kubernetes without helm` 
2. Consider renaming it to `kubernetes operator` so it will be more understandable for customers

Reply:
We plan on merging the docs update after the operator is GA. Sure I can rename it to Kubernetes Operator

Reply:
@urseberry What is the standard tab name that we should go with for the Datadog Operator?